src/style.css

```diff
@@ -1,3 +1,113 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+
 }
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+
+}
+@media (min-width: 1200px) {
+  .header {
+    width: 100%;
+  }
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.logo-moyo {
+  display: flex;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+}
+
+.nav__item {
+  line-height: 60px;
+  margin-left: 20px;
+  list-style: none;
+  text-transform: uppercase;
+  height: 60px;
```

Comment:
it is better to use padding

Reply:
Replaced padding-left with margin-right. But I don't know about the height, because in the task it is specified not to use the height in the header, but to set the height of the links.

Reply:
in the description of the task said that links should dictate the height of header, but not "set the height to the links", and mentors tried to tell you that you can achieve using paddings instead hardcoding their height